src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,207 @@
+package duke;
+
+import duke.command.AddTaskCommand;
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.CommandType;
+import duke.command.DeleteTaskCommand;
+import duke.command.EmptyCommand;
+import duke.command.ListCommand;
+import duke.command.SetDoneCommand;
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.Task;
+import duke.task.TodoTask;
```

Comment:
It's great that you listed all of the classes that you are importing instead of using a wildcard